web/styles/compose.css

```diff
@@ -304,14 +304,17 @@
 
 #compose-limit-indicator {
     font-size: 12px;
-    color: hsl(39deg 100% 50%);
+    color: var(--color-limit-indicator);
+    width: max-content;
+    padding: 3px;
 
-    /* Keep the limit indicator
-       just above the send button */
+    /* Keep the limit indicator just above
+       and aligned with the send button */
     margin-top: auto;
+    padding-left: 0;
```

Comment:
Tiny nit, but let's just modify the `padding` value to be `padding: 3px 3px 3px 0`